components/SiteHeader.tsx

```diff
@@ -1,21 +1,46 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Link from 'next/link'
+import classNames from 'classnames'
 import LogoIcon from '@/components/svg/LogoIcon'
 import ButtonLink from '@/components/ButtonLink'
+import HamburgerMenu from '@/components/HamburgerMenu'
 
 const AppHeader: React.FC = () => {
+  const [showMobileNav, setShowMobileNav] = useState(false)
+
+  function toggleMobileNav() {
+    setShowMobileNav(!showMobileNav)
```

Comment:
This is mostly a nit. I don't think in this simple case there is a problem, but below is the best practice when referencing previous state.

```suggestion
    setShowMobileNav(s => !s)
```

Reply:
Ah ha, TIL! Thanks.